src/org/jitsi/impl/neomedia/rtp/TransportCCEngine.java

```diff
@@ -361,5 +386,70 @@ private RTCPTransformer()
         }
     }
 
+    /**
+     * {@Link BITRATE_UPDATE_INTERVAL} is an interval used to
+     */
+    private static final int BITRATE_UPDATE_INTERVAL = 5;
```

Comment:
As a rule we define static fields at the top of the file, and we try to include the units in the name, e.g. BITRATE_UPDATE_INTERVAL_MS (if it is in milliseconds)